CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Released]
 
-### [v0.8.1] - 9/25/2020
+### [v0.8.2] - 9/25/2020
```

Comment:
A new section should be added, not removing the old one... just to keep track of what we changed and where.